java-pmd7/pom.xml

```diff
@@ -10,18 +10,42 @@
     <name>Sonar PMD7 jPinpoint rules for Java plugin</name>
 
     <properties>
-        <!-- read from plugin.properties: propfile.plugin.key and propfile.plugin.description -->
-        <!--suppress UnresolvedMavenProperty -->
-        <plugin.key>${propfile.plugin.key}</plugin.key>
-        <!--suppress UnresolvedMavenProperty -->
-        <plugin.description>${propfile.plugin.description}.</plugin.description>
-
+        <plugin.key>pmdjpinpoint</plugin.key>
+        <plugin.name>jPinpoint rules for Java</plugin.name>
+        <plugin.description>128 Rules for responsible Java programming built on PMD6, 79 on performance, 63 on sustainability, 32 on multi-threading, 10 on data mix-up.</plugin.description>
         <pmd.version>7.9.0</pmd.version>
-        <pmd.inputFileName>jpinpoint-rules-pmd7.xml</pmd.inputFileName>
-        <sonar.outputFileName>sonar-pmd-jpinpoint.xml</sonar.outputFileName>
+        <pmd.inputFileName>jpinpoint-rules.xml</pmd.inputFileName>
+        <sonar.outputFileName>sonar-pmd-jpinpoint-pmd7.xml</sonar.outputFileName>
```

Comment:
Here `-pmd7` in the file name?

Reply:
Not sure why indeed, I don't think I changed it. I guess we didn't change it, yet.